tests/kani/Spurious/storage_fixme.rs

```diff
@@ -0,0 +1,649 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Modifications Copyright Kani Contributors
+// See GitHub history for details.
+
+// Our handling of storage markers causes spurious failures in this test.
+// https://github.com/model-checking/kani/issues/3099
```

Comment:
How is this connected to #3079 ? The code below is taken from `std`, right?

Reply:
I expanded the description of the test to clarify where the code is extracted from.

Yes, it is connected to #3079 because we saw those spurious failures when trying to resolve this through enabling storage markers for `std` as well (#3080).